python/paddle/trainer_config_helpers/layers.py

```diff
@@ -225,6 +225,24 @@ def is_layer_type(type_name):
 
 
 class AggregateLevel(object):
+    """
+    PaddlePaddle supports three sequence types:
+
+    - :code:`SequenceType.NO_SEQUENCE` means the sample is not a sequence.
+    - :code:`SequenceType.SEQUENCE` means the sample is a sequence.
+    - :code:`SequenceType.SUB_SEQUENCE` means the sample is a nested sequence,
+      each timestep of which is also a sequence.
+
+    Accordingly, AggregateLevel supports two modes:
+
+    - :code:`AggregateLevel.EACH_TIMESTEP` means the aggregation acts on each
+      timestep of a sequence, both :code:`SUB_SEQUENCE` and :code:`SEQUENCE` will
```

Comment:
This actually aggregates over the whole sequence. Maybe change the name to TO_NON_SEUQNCE will be easier to understand.